client/my-sites/stats/stats-subscribers-chart-section/index.tsx

```diff
@@ -121,6 +121,33 @@ export default function SubscribersChartSection( {
 	const legendRef = useRef< HTMLDivElement >( null );
 	const translate = useTranslate();
 
+	const formatTimeTick = useCallback(
```

Comment:
I think maybe the naming of this constant stems from when we were using time ticks for realtime stats. Does this name still make sense? or should it be something else, like formatDateTick?

Reply:
Good point. I'll follow up with it...